invenio/templates/deployments/logstash.yaml

```diff
@@ -18,7 +18,9 @@ spec:
     spec:
       containers:
         - name: logstash
-          image: docker.elastic.co/logstash/logstash-oss:6.3.1
+          image: docker.elastic.co/logstash/logstash-oss:7.10.1
```

Comment:
I don't like that the image for logstash is hardcoded here, and the image for filebeat is read from the values. It would be more consistent to have them both in the same place.

Reply:
move it a variable in `values`

There is version 8 out already

Reply:
Maybe move it to `values.yaml` as @ntarocco  said. We follow that pattern for other services:

From `values.yaml`:

```yaml
rabbitmq:
  enabled: true
  image: "rabbitmq:3-management"
  existing_secret: false
  secret_name: "mq-secrets"
  default_password: "mq_password"
  celery_broker_uri: "amqp://guest:mq_password@mq:5672/"
```